packages/dso-toolkit/src/icons/asterisk.scss

```diff
@@ -2,8 +2,4 @@
 
 asterisk {
   color: colors.$grijs-50;
-}
-
-asterisk:grasgroen {
-  color: colors.$grasgroen;
-}
+}
```

Comment:
`yarn lint` rapporteert: `Code style issues found in 5 files. Run Prettier with --write to fix.` Waarvan dit er 1 is.